docs/reliability.md

```diff
@@ -0,0 +1,84 @@
+# Reliability
+
+In production the ingestion product aims to provide a Biweekly Uptime
+Percentage determined by the Reliability Target below. If a component does
+not meet that then a Stability Work Period should be assigned
+to each software engineer supporting the component.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Disclaimer and Purpose](#disclaimer-and-purpose)
+- [Reliability Target](#reliability-target)
+- [Definitions](#definitions)
+- [Exclusions](#exclusions)
+- [Additional Information](#additional-information)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Disclaimer and Purpose
+
+**This document is intended solely for those directly running, writing, and
+managing GCP Ingestion. It is not an agreement, implicit or otherwise, with any
+other parties.** This document is a prototype that should be treated as a goal
+that will require adjustments.
+
+The purpose of this document is first and foremost to encourage behavior that
+reduces Downtime. The secondary purpose is to establish clear expectations for
+how software engineers respond to Downtime.
+
+## Reliability Target
+
+| Component      | Biweekly Uptime Percentage |
+|----------------|----------------------------|
+| ingestion-edge | 99.99%                     |
+| ingestion-beam | 99.5%                      |
+
+## Definitions
+
+**"Downtime"** on ingestion-edge means for at least 0.1% of requests a status
+code lower than 500 was not successfully returned. On ingestion-beam it means
```

Comment:
Do we yet know how this will be measured and monitored? If so, let's add some detail / links to monitoring here. If not, let's file a follow up issue to figure that piece out.

Reply:
filed #296